src/ui/page/dataverse/dataverse.tsx

```diff
@@ -433,6 +438,36 @@ const dataverseItems: DataverseItemDetails[] = [
   }
 ]
 
+const selectionFilters = {
```

Comment:
These data should be fetched from the ontology, as it contains the controlled vocabularies and is therefore the unique source of truth. Will they be taken into account soon?

Reply:
Yes, the wiring to the ontology is carried out in the next iteration, in a dedicated PR.